documentation/styleguide-doc.md

```diff
@@ -0,0 +1,167 @@
+# Documentation Coding Style
+
+## Where are the docs?
+
+TL;DR version: [`docs`](https://github.com/StackExchange/dnscontrol/tree/master/docs) is the [marketing website](https://dnscontrol.org). [`documentation`](https://github.com/StackExchange/dnscontrol/tree/master/documentation) is the [docs.dnscontrol.org](https://docs.dnscontrol.org/) website. (Yes, the names are backwards!)
+
+**The two websites**
+
+1. <https://dnscontrol.org/>
+   * The main website
+   * Source code: [`docs`](https://github.com/StackExchange/dnscontrol/tree/master/docs)
+   * Mostly "marketing" for the project.
+   * Rarely changes.  Updated via GitHub "pages" feature.
+2. <https://docs.dnscontrol.org/>
+   * Project documentation
+   * Source code: [`documentation`](https://github.com/StackExchange/dnscontrol/tree/master/documentation)
+   * Users and developer documentation
+   * Changes frequently.  Updated via [GitBook](https://www.gitbook.com/)
+
+**The directory structure**
+
+Within the git repo, docs are grouped:
+
+* [`documentation/`](https://github.com/StackExchange/dnscontrol/tree/master/documentation): general docs
+* [`documentation/providers/`](https://github.com/StackExchange/dnscontrol/tree/master/documentation/providers/): One file per provider
+* [`documentation/functions/`](https://github.com/StackExchange/dnscontrol/tree/master/documentation/functions/): One file per `dnsconfig.js` language feature
+* [`documentation/assets/gcloud/`](https://github.com/StackExchange/dnscontrol/tree/master/documentation/assets/gcloud): Images for page Google Cloud DNS (PNGs only, please!)
```

Comment:
Please make this more generic.  For example...

```
* `documentation/assets/FOO/`: Images for page FOO (PNGs only, please)
```

Reply:
Fixed by https://github.com/StackExchange/dnscontrol/pull/2383/commits/2c8efa4001c86eeb7cf6f05cc5d1cc41f04be752. @tlimoncelli would you like to close this conversation after reviewing it?

Reply:
_<https://docs.dnscontrol.org/~/revisions/BRtPCFXEcJg89ztpW8cK/developer-info/styleguide-doc#where-are-the-docs>_